lib/frontend/templates/subtemplates/render_sentence.rs.html

```diff
@@ -0,0 +1,25 @@
+@use japanese::furigana::SentencePartRef;
+@type MyIterator<'a> = Vec<SentencePartRef<'a>>;
+@(iter: MyIterator)
```

Comment:
Oh and don't ask me about this one :p apparently ructe doesn't like `<>` in its type declarations??